linux_docker_resources/entry_point.sh

```diff
@@ -33,6 +33,12 @@ if [ "${ARCH}" != "aarch64" ]; then
         case "${CI_ARGS}" in
           *--connext-debs*)
             echo "Using Debian package of Connext"
+            if test -r /opt/rti.com/rti_connext_dds-6.0.1/resource/scripts/rtisetenv_x64Linux4gcc7.3.0.sh; then
+                echo "Sourcing RTI setenv script /opt/rti.com/rti_connext_dds-6.0.1/resource/scripts/rtisetenv_x64Linux4gcc7.3.0.sh"
+                . /opt/rti.com/rti_connext_dds-6.0.1/resource/scripts/rtisetenv_x64Linux4gcc7.3.0.sh
+            elif test -r /opt/rti.com/rti_connext_dds-5.3.1/resource/scripts/rtisetenv_x64Linux3gcc5.4.0.bash; then
+                echo "rti_connextdds_cmake_module will guess the location of Connext 5.3.1 so don't source anything."
```

Comment:
Out of curiousity, is it actively harmful to source the 5.3.1 setup script?  If not, I think it would be more consistent to just source it here (even if `rti_connextdds_cmake_module` may redo some work).

Reply:
Probably not. I was aiming for a minimal change. I think there's a fair bit of refactoring we could do to simplify the plumbing here if we take a step back and look.

Reply:
Oh! one minor speed bump is that we can't source the 5.3.1 script from a POSIX sh script without `sed`ing it first as we do in the release repository for connext_cmake_module (and which hasn't been ported to rti_connext_cmake_module due to the guessing behavior)

Reply:
That's fine, this isn't a blocking problem anyway.  Just curious about it.